### PR TITLE
[CLI] Add a command to open db connection.

### PR DESCRIPTION
**Pull request by wontruefree**
This adds a CLI command `connection` for quick commandline access.

### CODE REVIEW

**Comment by eliasjpr** [comment]
I think this is a great addition. But I am curious to how this will work given that I don't see specs. 

Will this allow execute queries? How will this work with Sqlite and MySQL?

File: src/amber/cli/commands/database.cr [hunk 1]
```diff
@@ -41,6 +41,11 @@ module Amber::CLI
 
       def run
         CLI.toggle_colors(options.no_color?)
+
+        if args.commands.empty?
```
Comment (eliasjpr) [inline src/amber/cli/commands/database.cr]: I don't know how I feel about defaulting to connect to the db when no arguments are present.

**Comment by faustinoaq** [comment]
Fixes #708

File: src/amber/cli/commands/database.cr (hunk 1 above)
Comment (robacarp) [reply src/amber/cli/commands/database.cr]: I'm assuming that this default would be invoked with simply `amber db`? If so, I see no problem with that.

**Pull request merged**